index.html

```diff
@@ -1,18 +1,64 @@
-<script>
-let name = document.location.pathname.slice(1);
+<!DOCTYPE html>
+<html lang="en">
 
-if (!name) {
-  console.log("no place name specfied");
-} else {
-  console.log("fetching place named '" + name + "'");
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="style.css" rel="stylesheet" />
+  <title>Document</title>
+  <link rel="stylesheet" href="https://unpkg.com/leaflet@1.6.0/dist/leaflet.css"
+    integrity="sha512-xwE/Az9zrjBIphAcBb3F6JVqxf46+CDLwfLMHloNu6KEQCAWi6HcDUbeOfBIptF7tcCzusKFjFw2yuvEpDL9wQ=="
+    crossorigin="" />
+  <script src="https://unpkg.com/leaflet@1.6.0/dist/leaflet.js"
+    integrity="sha512-gZwIG9x3wUXg2hdXF6+rVkLF/0Vi9U8D2Ntg4Ga5I5BZpVkVxlJWbSQtXPSiUTtC0TjtGOmxa1AJPuV0CPthew=="
+    crossorigin=""></script>
+</head>
 
-  fetch(name + '.json')
-    .then(function (response) {
-      return response.text();
-    })
-    .then(function (myText) {
-      console.log(myText);
-    });
-}
+<body>
+  <!--Title Header-->
+  <section class="main-container">
+    <!--Title Container-->
+    <section class="title-flex-container">
+      <p id="yelpington-flex"><a href="http://localhost:5000/" style="text-decoration: none">YELPINGTON...</a></p>
+    </section>
+    <!--Sidebar/Map-->
+    <section class="sidebar-map-container">
+      <!--Sidebar Container-->
+      <section class="sidebar-container">
+        <div id="restaurant-title">Restaurants</div>
+        <div class="restaurants" id="restaurant-list"></div>
+      </section>
+      <!--Map Container-->
+      <section class="map-container">
+        <div id="map"></div>
+        <script src="index.js"></script>
```

Comment:
All of the script tags that have code you wrote, you can bunch together at the bottom of your html document as the last tag inside your body, sort of like you have your script tag linking to main.js. These will work here, depending on what you are doing in your javascript, but if your browser is reading this, then it will try to execute the file when it is read, and if your file depends on elements in your html page and the page hasnt completely loaded yet, then your js may not work as intended, which is why these tags normally come at the bottom of your page.